lib/datadog/appsec/contrib/devise/ext.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Devise
+        # Devise integration constants
+        module Ext
+          APP = 'devise'
+          DISABLED_MODE = 'disabled'
+          EXTENDED_MODE = 'extended'
```

Comment:
`safe` does not appear here, so this doesn't feel exhaustive. It feels like it should be an exhaustively listing enum.